code/workspaces/infrastructure-api/src/controllers/stackController.spec.js

```diff
@@ -142,6 +146,13 @@ describe('Stack Controller', () => {
         .then(() => expectValidationError('visible', 'visible must be specified for sites'));
     });
 
+    it('should validate the version field', () => {
+      const requestBody = mutationRequestBody();
+      requestBody.version = 'invalid-version';
+      return createValidatedRequest(requestBody, stackController.createStackValidator)
+        .then(() => expectValidationError('version', 'valid version must be specified'));
+    });
```

Comment:
Do there also need to be tests to make sure that it validates correctly when:
* a version is not supplied in the call;
* and the version is valid?

I don't know whether any of this is covered off by the existing tests.